From e43df5e027b721f65989d67fd8f53656fd181b95 Mon Sep 17 00:00:00 2001
From: Chris Renshaw <osm0sis@outlook.com>
Date: Sat, 9 May 2020 06:53:32 -0300
Subject: [PATCH 3/9] init: add vendor.*  keys to spoof safetynet

aswinas@pixysos: add some more props from magisk hide to userspace hack by arter97

Change-Id: Ib6ad1df4582a2f8333b0cbf650e79e947f6576d0
---
 init/property_service.cpp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index d3c51ba0b9..63895b68e6 100644
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -812,6 +812,10 @@ static const char *snet_prop_key[] = {
 	"ro.build.keys",
 	"ro.build.tags",
 	"ro.system.build.tags",
+	"ro.vendor.boot.warranty_bit",
+	"ro.vendor.warranty_bit",
+	"vendor.boot.vbmeta.device_state",
+	"vendor.boot.verifiedbootstate",
 	NULL
 };
 
@@ -829,6 +833,10 @@ static const char *snet_prop_value[] = {
 	"release-keys", // ro.build.keys
 	"release-keys", // ro.build.tags
 	"release-keys", // ro.system.build.tags
+	"0", // ro.vendor.boot.warranty_bit
+	"0", // ro.vendor.warranty_bit
+	"locked", // vendor.boot.vbmeta.device_state
+	"green", // vendor.boot.verifiedbootstate
 	NULL
 };
 
-- 
2.34.1

